### ISSUE TITLE
Need an entrance of Mobile

### ISSUE THREAD

**Issue by NHZlX** [PaddlePaddle/PaddlePaddle.org#134]
There is a need for an entrance of Mobile. The following image shows the demo. 
![1556b46af67b7f573f0d9e05c98bada7](https://user-images.githubusercontent.com/5595332/31486351-d2ce9944-aefc-11e7-8109-370bd1ac22dd.png)
In the left column, we need a first level title for Mobile, the right side corresponding text should be the content of [this](https://github.com/PaddlePaddle/Mobile/blob/master/README.md). The image is just a demo, may be there is a better design.

**Comment by varunarora** [comment]
Thanks for pulling the repo and running it locally!

It would be awesome to have that mobile building doc in, but if it is going to reside in the `Mobile` repo, it would add a deployment / building dependency on a 4th repo (after `models`, `book` and ofcourse `paddle`).

Would you be open to adding this page into the primary Sphinx documentation in `doc`? Having a duplicate copy is the obvious suggestion, but I guess something more efficient would be to just link to the page from `paddle` repo on the README of `Mobile`.

**Comment by jetfuel** [comment]
@varunarora @cs2be Do we have a plan to proceed with this task? Will our tool able to transform the initial README.md to an HTML file?

**Comment by NHZlX** [comment]
@varunarora  I did not pull the repo and run it, i just modify the page on line. We just want the user to know there is a [Mobile repo](https://github.com/PaddlePaddle/Mobile), so, we need an entrance in the Documentation.

**Comment by varunarora** [comment]
@NHZlX Aha! Nice one.

Great - so is it possible for you to add this mobile building page to the main `/Paddle` doc?

@jetfuel Absolutely we can, but we should try to avoid introducing the dependency of another repo for a short tutorial, if we can.

**Comment by varunarora** [comment]
@NHZlX We have decided to work on pulling contents from the `Mobile` repo directly, so that question for you is invalid now.

**Comment by varunarora** [comment]
Progress in #185

**Comment by varunarora** [comment]
This resolution for this issue transitioned for now. In summary, the original goal was to find a way to add the documentation for mobile to in `Paddle` repos `doc`. I then suggested pulling the contents from the `Mobile` repo during CI. We finally resorted to breaking that chain of dependency to allowing any arbitrary raw markdown references in sitemaps.

Arbitrary markdown from Github now supported.
